builtin/providers/aws/resource_aws_api_gateway_integration.go

```diff
@@ -119,11 +132,12 @@ func resourceAwsApiGatewayIntegrationCreate(d *schema.ResourceData, meta interfa
 		IntegrationHttpMethod: integrationHttpMethod,
 		Uri: uri,
 		// TODO reimplement once [GH-2143](https://github.com/hashicorp/terraform/issues/2143) has been implemented
-		RequestParameters:  aws.StringMap(parameters),
-		RequestTemplates:   aws.StringMap(templates),
-		Credentials:        credentials,
-		CacheNamespace:     nil,
-		CacheKeyParameters: nil,
+		RequestParameters:   aws.StringMap(parameters),
+		RequestTemplates:    aws.StringMap(templates),
+		Credentials:         credentials,
+		CacheNamespace:      nil,
+		CacheKeyParameters:  nil,
+		PassthroughBehavior: aws.String(d.Get("passthrough_behavior").(string)),
```

Comment:
As this is an optional value and has no default value, we should check to see if it is set before adding to the params